apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -1184,6 +1252,10 @@ cm_p2p_headers() ->
 	{ok, Config} = application:get_env(arweave, config),
 	add_header(<<"x-cm-api-secret">>, Config#config.cm_api_secret, p2p_headers()).
 
+pool_client_headers() ->
+	{ok, Config} = application:get_env(arweave, config),
+	add_header(<<"x-pool-api-key">>, Config#config.pool_api_key, p2p_headers()).
```

Comment:
I forget where we landed on TLS/SSL - right now by default all peers communicate unencrypted via HTTP. If that's the case I think using a shared secret will be a problem.

Reply:
We plan to set up https on the proxy. The support for https requests on the client side is in place.

Reply:
Got it. 👍